components/camel-debezium/pom.xml

```diff
@@ -38,6 +38,7 @@
         <module>camel-debezium-mysql</module>
         <module>camel-debezium-postgres</module>
         <module>camel-debezium-sqlserver</module>
+        <module>camel-debezium-oracle</module>
```

Comment:
Can you put this in a sorted A..Z position

Reply:
done